iscservice/ut/service_ut.c

```diff
@@ -172,10 +170,10 @@ static void isc_ut_server_stop(void)
 
 static void isc_ut_client_start(void)
 {
-	int rc;
+	int                 rc;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated